woocommerce/woocommerce
# Fix increasing & decreasing sale price in product bulk edit

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/43149



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `WooCommerce > Products`.
2. Select a product from the list.
3. Click on `Bulk actions`, select `Edit`, and click the `Apply` button.
4. Click on the `Sale` input and select `Decrease existing sale price by (fixed amount or %):`.
5. Enter a percentage sale (like 10%) click `Update`, and check the product is on sale and the price is decreased in the percentage you entered.
6. Update the price and remove the sale price.
7. Repeat steps 2-5 with the `Increase existing sale price by (fixed amount or %):` option.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Product bulk edit: fix increasing & decreasing sale price when there was no previous sale.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
